### PR TITLE
doc: included previous known issues

### PR DESCRIPTION
**Pull request by annwoj**
Added previous known issues per release.
Added toggle extension.

### CODE REVIEW

**Review by ktaborowski — CHANGES_REQUESTED** [review]
Please add dev to v1.14.5 before merge (another comments are just comment)

File: doc/sidewalk_release_notes.rst [hunk 1]
```diff
@@ -20,6 +20,93 @@ Dynamic FSK-WAN Profile switching from 2 to 1 is prohibited.
...
+
+.. toggle::
+
+    * KRKNWK-14583: Bus fault after flash, before manufacturer data flash
+        For LoRa sample, when there is no manufacturer hex, the device throws a hard fault after being powered up.
+        Proper error handling will be implemented, but the temporary solution is to write manufacturing hex to the device and reset it.
+
+    * KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
+        The non-resolvable private address (NRPA) cannot be set in the connectable mode for Bluetooth LE.
+        Currently there is no workaround for this issue.
```
[the remark this answers is not in the document]
Comment (annwoj) [reply doc/sidewalk_release_notes.rst]: thanks for checking! removed

File: doc/conf.py
[the remark this answers is not in the document]
Comment (annwoj) [reply doc/conf.py:1]: corrected

**Review by ktaborowski — APPROVED** [review]
looks good :)

**Pull request merged**